extensions/sql-migration/images/background.svg

```diff
@@ -0,0 +1,38 @@
+<svg width="753" height="242" viewBox="0 0 753 242" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
could you please attach screenshots for dark and high contrast modes?

Reply:
![image](https://user-images.githubusercontent.com/6816294/108259325-cfd49700-7115-11eb-85bf-c67487205e60.png)